jbehave-support-core/src/main/java/org/jbehavesupport/core/support/RequestFactory.java

```diff
@@ -9,11 +9,7 @@
 import static org.springframework.util.StringUtils.hasText;
 
 import java.beans.PropertyDescriptor;
-import java.lang.reflect.Field;
-import java.lang.reflect.Method;
-import java.lang.reflect.Modifier;
-import java.lang.reflect.ParameterizedType;
-import java.lang.reflect.Type;
+import java.lang.reflect.*;
```

Comment:
Is it wanted or some unwanted change made by IDE?

Reply:
Changed back 🙂